app/views/globalPromotions.html

```diff
@@ -63,22 +63,178 @@ <h2 translate>{{$ctrl.getMinistryName()}} Promotions</h2>
           New Promotion
         </button>
       </div>
-      <div class="table-responsive" ng-if="$ctrl.promotions.length > 0">
+
+      <div
+        class="row well well-full"
+        ng-if="$ctrl.promotions.length > 0 && !$ctrl.editingPromotion"
+      >
+        <div class="col-md-6 col-sm-6">
+          <div class="col-md-6 col-sm-10 col-xs-12 padding-left-0">
```

Comment:
Do you think we should put the two filters next to each other on larger screens since they are related? There's a lot of space between them how it is now.

Reply:
I'm open to that! I don't have a strong preference. Feel free to change it if you'd like.